src/app/public/modules/phone-field/phone-field.component.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  OnDestroy
+} from '@angular/core';
+
+require('intl-tel-input/build/js/utils');
+require('intl-tel-input/build/js/intlTelInput');
+
+/**
+ * NOTE: We can not type these due the the current @types/intl-tel-input version having an
+ * undeclared type which causes linting errors.
+ */
+declare var intlTelInputUtils: any;
+declare var intlTelInputGlobals: any;
+
+import {
+  SkyCountryData
+} from './types';
+
+@Component({
+  selector: 'sky-phone-field',
+  templateUrl: './phone-field.component.html',
+  styleUrls: ['./phone-field.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyPhoneFieldComponent implements OnDestroy {
+
+  public countryData: SkyCountryData[];
+
+  public defaultCountryData: SkyCountryData;
+
+  public disabled: boolean;
```

Comment:
Booleans need deliberate initial values to avoid the "falsyness" of `undefined` value.
```
public disabled = false;
```

Reply:
Done